docker/docker-compose.yml

```diff
@@ -1,5 +1,8 @@
 version: "2"
 
+# IMPORTANT NOTE: For the web container we use exposing the code via a volume, not via
```

Comment:
Not quite true - [the dockerfile](https://github.com/metabrainz/listenbrainz-server/blob/747118854aa91c877c974ed95b1710ec3c8988e4/docker/Dockerfile#L38) also uses `COPY` to add the code. The use of volumes is for during development - e.g., if you have flask autoreloader configured to restart on file change you need your most up-to-date version of the code always in the container (otherwise you would need to `docker-compose build / down / up` after every file change). I'd say this instead:

```
IMPORTANT NOTE: Volume paths mounted on containers are relative to the directory that this file is in (`docker/`)
and so probably need to start with `../` to refer to a directory in the main code checkout
```